src/coreclr/jit/fginline.cpp

```diff
@@ -65,16 +79,19 @@ unsigned Compiler::fgCheckInlineDepthAndRecursion(InlineInfo* inlineInfo)
 // Return Value:
 //    True if the inline is recursive and should be disallowed.
 //
-bool Compiler::IsDisallowedRecursiveInline(InlineContext* ancestor, InlineInfo* inlineInfo)
+bool Compiler::IsDisallowedRecursiveInline(InlineContext* ancestor, InlineInfo* inlineInfo, bool* triviallyRecursive)
```

Comment:
not clear to me why this method has to expose a parameter, can it just do everything inside?

Reply:
Resolved.